.github/workflows/build.yml

```diff
@@ -114,40 +120,37 @@ jobs:
       run: |
         export QT_SELECT=qt5
         echo ======= SYSTEM INFO ========
-        uname -a; gcc --version | grep "gcc"; python --version; qmake-qt5 --version
+        which python 2>/dev/null && export PYTHON_VERSION_SUFFIX= || export PYTHON_VERSION_SUFFIX=3
+        uname -a; gcc --version | grep "gcc"; python${PYTHON_VERSION_SUFFIX} --version; qmake-qt5 --version
         echo ============================
         qmake-qt5 -r PythonQt.pro CONFIG+=${{ matrix.configuration }} \
-          PYTHON_VERSION=$(python --version | cut -d " " -f 2 | cut -d "." -f1,2) \
-          PYTHON_DIR=$(which python | xargs dirname | xargs dirname)
+          PYTHON_VERSION=$(python${PYTHON_VERSION_SUFFIX} --version | cut -d " " -f 2 | cut -d "." -f1,2) \
+          PYTHON_DIR=$(which python${PYTHON_VERSION_SUFFIX} | xargs dirname | xargs dirname)
         make -j 2 && make check TESTARGS="-platform offscreen"
-      
+
     - name: Generate Wrappers
       run: |
         # workaround to allow to find the Qt include dirs for installed standard qt packages
         mkdir /usr/include/qt5ln; ln -s /usr/include/qt5 /usr/include/qt5ln/include
         export QTDIR=/usr/include/qt5ln
         cd generator
         ./pythonqt_generator
-        
+
     - name: Upload Wrappers
       uses: actions/upload-artifact@v3
       with:
-        name: wrappers_centos7
+        name: wrappers_${{ matrix.container_os }}-${{ matrix.container_os_version }}_${{ matrix.configuration }}
         path: generated_cpp
 
   macOS:
     strategy:
       fail-fast: false
       matrix:
-       macos-version:  ['10.15']
-       python-version: ['2.7']
+       macos-version:  ['11']
+       python-version: ['3.6']
```

Comment:
Hopefully, 3.6 will be supported a bit longer by GH Actions, but eventually it will also go... Well we can deal with it when it becomes a problem.